core/src/main/java/bio/terra/pearl/core/service/notification/email/EnrolleeEmailService.java

```diff
@@ -158,7 +156,7 @@ public boolean shouldSendEmail(Trigger config,
                     enrolleeContext.getEnrollee().getShortcode(), config.getId(), config.getPortalEnvironmentId());
             return false;
         }
-        if (enrolleeContext.getProfile().isDoNotEmailSolicit() && config.getTaskType().equals(TaskType.OUTREACH)) {
+        if (enrolleeContext.getProfile().isDoNotEmailSolicit() && Objects.nonNull(config.getTaskType()) && config.getTaskType().equals(TaskType.OUTREACH)) {
```

Comment:
Drive by fix: task type is not technically required, so this could throw null pointer errors.